src/frontend/src/icp/components/send/IcSendTokenWizard.svelte

```diff
@@ -131,28 +131,31 @@
 				ckErc20ToErc20MaxCkEthFees
 			};
 
+			const trackAnalyticsOnSendComplete = async () => {
```

Comment:
didn't Lint asked you to remove the `{}` parenthesis? strange...

Reply:
No it didn't. I added those because in the library I got a different setup. We should maybe align the rules.

Which one is the particular eslint rule? That way I can copy it?

Reply:
Ah it's probably because it's in a Svelte component script tag?

Reply:
Anyway I need the `{}` because otherwise the function does not return `Promise<void>` but the result of `allSettled` here.

Reply:
all good! no worries! it is indeed because you are not returning it, it is void, so the linter worked. [The rule](https://eslint.org/docs/latest/rules/arrow-body-style#as-needed) is

	`'arrow-body-style': ['warn', 'as-needed']`